### ISSUE TITLE
🐛 Bug: Poor Color Contrast on Article Quotes in Dark Mode

### ISSUE THREAD

**Issue by DarhkVoyd** [json-schema-org/website#771]
### Describe the bug

The color contrast on the quotes in articles is not adequate when viewed in dark mode. This issue has been identified on our latest [blog post](https://json-schema.org/blog/posts/json-schema-ambassadors#the-time-is-right-we-are-growing), as shown in the attached screenshot. According to the Web Content Accessibility Guidelines (WCAG), the minimum contrast ratio for normal text is 4.5:1 and for large text is 3:1. The current contrast ratio does not meet these guidelines, making it difficult for users to read the content.

Check out the MDN guidelines [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast)

### Steps To Reproduce

1. Open the [article page](https://json-schema.org/blog/posts/json-schema-ambassadors#the-time-is-right-we-are-growing) in dark mode.
2. Inspect the quotes in the article.

### Expected Behavior

The contrast ratio between the text and background should be enough that it is readable.
It should be easy to do better in this instance.

### Screenshots

![screenshot_2024-06-20_at_15 30 59](https://github.com/json-schema-org/website/assets/77478658/1b63e250-de09-49ad-a7b3-238452ff5661)

### Device Information [optional]

```markdown
- OS: MacOS
- Browser: Safari
- version:
```

### Are you working on this issue?

No

### Do you think this work might require an [Architectural Decision Record (ADR)]? (significant or noteworthy)

No
